frontend/appflowy_flutter/lib/plugins/trash/src/trash_cell.dart

```diff
@@ -62,10 +62,8 @@ class TrashCell extends StatelessWidget {
 
   String dateFormatter($fixnum.Int64 inputTimestamps) {
     final outputFormat = DateFormat('MM/dd/yyyy hh:mm a');
-    final date = DateTime.fromMillisecondsSinceEpoch(
-      inputTimestamps.toInt() * 1000,
-      isUtc: true,
-    );
+    final date =
+        DateTime.fromMillisecondsSinceEpoch(inputTimestamps.toInt() * 1000);
```

Comment:
Consider adding an extension on DateTime called `fromSecondsSinceEpoch` that takes the inputTimeStamp and does the multiplication. That would consolidate this "magic" number.

Reply:
You can always make it a static method or getter in the extension and not a constructor.

Reply:
Hey @Xazin, thanks for the suggestion. Static method works fine:

```
void main() {
  final timestamp = DateTime.now().millisecondsSinceEpoch ~/ 1000;
  final time = DateTimeExtension.fromSecondsSinceEpoch(timestamp);
  print(time);
}

extension DateTimeExtension on DateTime {
  static DateTime fromSecondsSinceEpoch(int seconds) =>
      DateTime.fromMillisecondsSinceEpoch(seconds * 1000);
}
```

But if I had to choose between the original and this one, I'd much prefer the original with comments since this introduces a DateTimeExtension out of nowhere, which doesn't improve readability much. As for the getter method, do you have an example code snippet? I don't quite get what you mean.

Reply:
Ah yes, I see. 

Leave it as is for now, maybe extract `1000` to a const `const millisToSeconds = 1000` or a function `millisToSeconds(int milliseconds) => milliseconds ~/ 1000`, or both...

I'm not sure if we will have support for the above in the near future, I doubt it though.

Reply:
Alrighty then, I'm going to resolve this thread